S2VX.Game/Editor/ToolState/HoldNoteToolState.cs

```diff
@@ -24,21 +29,49 @@ public class HoldNoteToolState : S2VXToolState {
         }
 
         public override bool OnToolClick(ClickEvent _) {
-            var note = new EditorHoldNote {
-                Coordinates = Editor.MousePosition,
-                HitTime = Time.Current,
-                EndTime = Time.Current + 1000    // TODO: #216 be able to change hold duration
-            };
-            Editor.Reversibles.Push(new ReversibleAddHoldNote(Story, note));
+            if (!IsRecording) {
+                HitTime = Time.Current;
+                Coordinates = Editor.MousePosition;
+            } else {
+                var endTime = Time.Current;
```

Comment:
Should we enforce that endTime > HitTime?

Reply:
That's a good point, I'll see how to address this

Reply:
Made a change so that if you click a second time and the end time is <= hit time, the hold note is not created, and you go back to the initial note placement phase. This follows osu!'s way of handling spinners.